### ISSUE TITLE
Core Theory: Accessibliity / ARIA

### ISSUE THREAD

**Issue by christopherreay** [grassrootseconomics/public-dashboard#10]
### Aria links and documentation
* since we are using bootstrap, and #9 we are able to directly encode accessibility standards into our `HTML` and `attributes`, and maintain the highest standards of accessibility.
* WCAG21 https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
* How to lay out landmarks in a page for a screenreader
  * https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11
  * 557adeb167ace3a0a952554b602c903c43c5ba49

### Developer tools for accessibility
* both chrome and firefox have reasonably developed tools for accessibility analysis, showing the acccessiblity tree of the website (rather than the DOM tree). They are not the same.
* NVDA is a free screen reader tool created by two blind programmers
  * here are some useful instructions as to how to use NVDA
    * https://dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts

### links/docs
* Here we will link documents that have been used to complete the accessibility tree of the site.
* #### How to manage modal menus
  * Keyboard behaviour, other minimum requirements
    * https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role
  * Focusing on the first menu item
    * using javascript `element.focus()` method will sometimes not visibly show the selection of the item, use `element.focus({focusVisibility: true})` option from here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus
* #### Different types of Button
  * https://www.deque.com/blog/accessible-aria-buttons/
  * how to make a bootstrap button look like a link
    * https://stackoverflow.com/questions/19981949/how-to-make-a-button-in-bootstrap-look-like-a-normal-link-in-nav-tabs
    * tl;dr; use classes `btn btn-link nav-link`, awesome :)
